### DISCUSSION TITLE
Support Lucia auth as an alternative auth solution

### DISCUSSION THREAD

**Discussion by ThimoDEV** [vikejs/bati#271] [Integration request]
### Name

Lucia auth

### Homepage

https://lucia-auth.com/

**Comment by magne4000** [comment]
Implemented in https://github.com/vikejs/bati/pull/310

**Comment by magne4000** [comment]
Removed in #530

> Lucia v3 has been deprecated. Lucia is now a learning resource for implementing sessions and more.
